### ISSUE TITLE
Activity Names, Slugs and Suggestion IDs etc not coming through in Kibana

### ISSUE THREAD

**Issue by eddieTomo** [remihellotomo/Tomo-Office#17]
## Expected behaviour
Should be able to query Elasticsearch logs in Kibana for activity, using
`context.key.habit` for activity uuid (which is same as in Ops dashboard)
`context.current_habit_script_name` for script associated with activity
`context.current_habit_slug` for slug of activity
`context.current_habit_name` for activity name

## Actual behaviour
No data exists in for these slugs in `tomo-dev` since 4th Oct 2017, or in `tomo-prod` since 10 November 2017.  Before this date data exists in both indexes and parses correctly so assume it is an issue with the input feed.

## Steps to reproduce
Search for existence of any of the above fields in Kibana, e.g. `context.key.habit:*`

**Comment by eddieTomo** [comment]
confirmed still happening as of dev build 5.3.26

**Comment by eddieTomo** [comment]
I have created a `habitmapper` script in Elasticsearch to work around this problem as a temporary. But it must be run manually before each analysis is generated.

**Comment by remihellotomo** [comment]
@eddieTomo - any reason why this ticket deals with activities and yet all the tags are 'habit'. Or is this because our ops/dev has the activities listed as habits? I just worry about confusion, ie, : '`context.current_habit_name` for activity name seems unclear.  I know that we define habits in the app as differently to what they are in our back system, but I really would like our back system to be as close to the app experience or else we'll continue to use old terminology with old meaning.  I'd be keen to understand what the impact would be to change 'habit' to 'activity' in ops/dev and with the UUIDs. I'm sure it's quite large.  Which is a shame, because we'll just have this confusion continued.

**Comment by eddieTomo** [comment]
@remihellotomo yes, this is because in the back end, 'activities' are called 'habits'.  in theory it's just a find & replace to replace everything in both dialogues and code but i think it's a bigger task than just that as we'll no doubt introduce a  lot of bugs.

Yes, in time, I would like to have everything named the same, it's currently in the `post-LSBU2` list of issues though
